src/Files.App/Actions/Navigation/OpenDirectoryInNewTabAction.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+	internal class OpenDirectoryInNewTabAction : ObservableObject, IAction
+	{
+		private readonly IContentPageContext context;
+
+		private readonly MainPageViewModel _mainPageViewModel;
+
+		public string Label
+			=> "OpenDirectoryInNewTab".GetLocalizedResource();
+
+		public string Description
+			=> "OpenDirectoryInNewTabDescription".GetLocalizedResource();
+
+		public RichGlyph Glyph
+			=> new(opacityStyle: "ColorIconOpenInNewTab");
+
+		public bool IsExecutable
+			=> context.HasSelection;
```

Comment:
And this as long as only folders are selected.

Reply:
Fixed